File: Go/SiskiRem.go
package main

import "fmt"

func main() {
	var RamBreastSize float64 = 2.0
	var Ram Women = Women{RamBreastSize}
	var Rem Women = Women{RamBreastSize * 1.5}
	fmt.Printf("(.)(.) - Это сиськи Рем. Они больше, чем у Рам. Размер:%f\n", Rem.BoobSize)
	fmt.Printf("(.)(.) - Это сиськи Рам. Они меньше, чем у Рем. Размер: %f\n", Ram.BoobSize)
}

type Women struct {
	BoobSize float64
}

File: Lua/siskirem.lua
Female = {
    BoobSize = 0
}
Female.__index = Female

function Female:new()
    return setmetatable({}, self)
end

Ram = setmetatable({}, { __index = Female })
Ram.__index = Ram

Ram.BoobSize = 2

Rem = setmetatable({}, { __index = Ram })
Rem.__index = Rem

Rem.BoobSize = Ram.BoobSize * 1.5

print("Это сиськи Рем, они больше, чем у Рам. Размер: " .. Rem.BoobSize .. "\nЭто сиськи Рам, они меньше, чем у Рем. Размер: " .. Ram.BoobSize)
